.github/workflows/create_eval_villian_update.yml

```diff
@@ -34,6 +34,7 @@ jobs:
         ## If there are changes: comment, commit, PR
         if ! git diff-index --quiet HEAD --; then
           ./gradlew :addOns:evalvillain:updateChangelog --change="- Updated with new version of Eval Villain."
+          ./gradlew :aO:webdrivers:evalvillain:prepareRelease
```

Comment:
Needs tweak to remove the parent project.

Reply:
Oops, TY